staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/capmanagers.go

```diff
@@ -78,15 +77,15 @@ func (f *capManagersManager) capUpdateManagers(managed Managed) (newManaged Mana
 
 	// If we have more than the maximum, sort the update entries by time, oldest first.
 	sort.Slice(updaters, func(i, j int) bool {
-		iTime, jTime, nTime := managed.Times()[updaters[i]], managed.Times()[updaters[j]], &metav1.Time{Time: time.Time{}}
-		if iTime == nil {
-			iTime = nTime
+		iTime, jTime, iSeconds, jSeconds := managed.Times()[updaters[i]], managed.Times()[updaters[j]], int64(0), int64(0)
```

Comment:
This sort routine doesn't actually matter for this bug, since it combines the oldest ones, it's fine if new ones have extra precision.